### ISSUE TITLE
Showing at both 11AM and 12PM

### ISSUE THREAD

**Issue by jdbrignone** [alyssadev/fitbit-wordclock#1]
Great simplistic watch face! I have found that it never says “ELEVEN” and instead it refers to “MIDDAY” for both 11am and 12pm. I’m guessing it could be something with the if statement but I thought I’d bring it up.

**Comment by alydevs** [comment]
You're right, I see the issue. Unfortunately I don't own a fitbit anymore and I don't have any easy way to troubleshoot, fix, and push an update for this watchface at the moment, sorry.
